kms/api/Kms.Samples/EncryptAsymmetric.cs

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START kms_encrypt_asymmetric]
+
+using Google.Cloud.Kms.V1;
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+public class EncryptAsymmetricSample
+{
+    public byte[] EncryptAsymmetric(
+      string projectId = "my-project", string locationId = "us-east1", string keyRingId = "my-key-ring", string keyId = "my-key", string keyVersionId = "123",
+      string plaintext = "Sample message")
```

Comment:
The idea that cryptographic plaintexts are bytes, as opposed to text, is often confusing to to cryptography novices.  It pushes people down all kinds of strange roads (e.g. "I have bytes, but they need text, I guess I will base64-encode...")

I'd reserve the word 'plaintext' to refer to a byte array, and call out the transformation you're doing, like:

```
public byte[] EncryptAsymmetric(..., string message = "Sample message")
{
    // Cryptographic plaintexts are always byte arrays. Convert our message into UTF-8 bytes.
    byte[] plaintext = Encoding.UTF8.GetBytes(message);
    ...
```

Reply:
Done